addons/payment/static/src/xml/payment_post_processing.xml

```diff
@@ -3,142 +3,72 @@
-                                <t t-esc="tx['reference']"/>
-                                <span class="badge float-end"><t t-esc="tx['amount']"/> <t t-esc="tx['currency_code']"/></span>
-                            </h4>
-                            <small class="list-group-item-text">
-                                <t t-if="tx['display_message']">
-                                    <!-- display_message is the content of the HTML field associated
-                                    with the current transaction state, set on the provider. -->
-                                    <t t-out="tx['display_message']"/>
-                                </t>
-                                <t t-else="">
-                                    Your payment is in pending state.<br/>
-                                    You will be notified when the payment is fully confirmed.<br/>
-                                    Click here to be redirected to the confirmation page.
-                                </t>
-                            </small>
-                        </a>
-                    </t>
-                    <!-- Authorized transactions -->
-                    <t t-foreach="tx_authorized" t-as="tx">
-                        <a t-att-href="tx['landing_route']" class="list-group-item">
-                            <h4 class="list-group-item-heading mb5">
-                                <t t-esc="tx['reference']"/>
-                                <span class="badge float-end"><t t-esc="tx['amount']"/> <t t-esc="tx['currency_code']"/></span>
-                            </h4>
-                            <small class="list-group-item-text">
-                                <t t-if="tx['display_message']">
-                                    <!-- display_message is the content of the HTML field associated
-                                    with the current transaction state, set on the provider. -->
-                                    <t t-out="tx['display_message']"/>
-                                </t>
-                                <t t-else="">
-                                    Your payment has been received but need to be confirmed manually.<br/>
-                                    You will be notified when the payment is confirmed.
-                                </t>
-                            </small>
-                        </a>
-                    </t>
-                </div>
+            <a t-att-href="landing_route" class="alert-link ms-auto text-nowrap">Skip<i class="oi oi-arrow-right ms-1 small"/></a>
+        </div>
+
+        <div class="o_payment_details o_cc2 row row-cols-1 row-cols-md-2 row-cols-lg-4 mx-0 mb-3 py-2 rounded">
+            <div class="col my-3">
+                <label for="form_reference" class="d-block small text-muted">
+                    Reference
+                </label>
+                <a t-att-href="landing_route"
```

Comment:
You are right, especially that "skip" button has same functionality. Since not always we want to give paying user access to document I think it's better to just leave reference as info not a link.